src/utils.js

```diff
@@ -1,11 +1,8 @@
 'use strict'
-
-const stream = require('stream')
-
-const { kHeaders } = require('./symbols')
+const { Stream } = require('stream')
 
 function isReadable (obj) {
-  return obj instanceof stream.Stream && typeof obj._read === 'function' && typeof obj._readableState === 'object'
+  return obj instanceof Stream && typeof obj._read === 'function' && typeof obj._readableState === 'object'
```

Comment:
This is a bit overly strict. Usually I do the check more in terms of "readablelike", i.e. check for `on`, `resume` and `pause`.

Reply:
I'm actually going to be changing this to just check for asyncIterator - since this is the common denominator for WHATWG ReadableStream and Node.js ReadableStream